ui/src/actions/sampleview.js

```diff
@@ -492,73 +421,6 @@ export function sendStartClickCentring() {
   };
 }
 
-export function sendZoomPos(level) {
-  return (dispatch) => {
-    fetch('/mxcube/api/v0.1/sampleview/zoom', {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-      body: JSON.stringify({ level }),
-    });
-  };
-}
-
-export function sendLightOn(name) {
-  return (dispatch) => {
-    dispatch(saveMotorPosition(name, true));
-    fetch(`/mxcube/api/v0.1/sampleview/${name.toLowerCase()}on`, {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-    }).then((response) => {
-      if (response.status >= 400) {
-        dispatch(showErrorPanel(true, 'Server refused to turn light on'));
-      }
-    });
-  };
-}
-
-export function sendLightOff(name) {
-  return (dispatch) => {
-    dispatch(saveMotorPosition(name, false));
-    fetch(`/mxcube/api/v0.1/sampleview/${name.toLowerCase()}off`, {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-    }).then((response) => {
-      if (response.status >= 400) {
-        dispatch(showErrorPanel(true, 'Server refused to turn light off'));
-      }
-    });
-  };
-}
-
-export function sendStopMotor(motorName) {
-  return () => {
-    fetch(`/mxcube/api/v0.1/sampleview/${motorName}/stop`, {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-    }).then((response) => {
-      if (response.status >= 400) {
-        throw new Error('Server refused to stop motor');
-      }
-    });
-  };
-}
-
```

Comment:
None of these seem to be used. Some of the endpoints still seem to exist in `samplecentring.py`, though.

Reply:
Great !